.github/workflows/CI_publish_snapshots.yml

```diff
@@ -28,7 +28,10 @@ jobs:
                     packageUser: ${{ secrets.BALLERINA_BOT_USERNAME }}
                     packagePAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
                 run: |
-                    ./gradlew clean build -x createJavadoc --scan publish --continue --rerun-tasks
+                    ./gradlew clean build ::createCodeCoverageReport -x createJavadoc --scan publish --continue --rerun-tasks
```

Comment:
```suggestion
                    ./gradlew clean build createCodeCoverageReport -x createJavadoc --scan publish --continue --rerun-tasks
```

Reply:
commit id is 47b7a88d363dc68ba4f0c7b3dce7c14ed1d168e3